html_css/v2/main-course/responsive-css/media-queries.md

```diff
@@ -0,0 +1,88 @@
+
+This does _not_ mean, however that you should just start your project with a media queries for each device. Each project is going to have different requirements based on the design you're trying to achieve. As mentioned above, try to limit your breakpoints to just what you _need_. With many relatively simple layouts, you can get by with only one mobile-centric breakpoint somewhere around `500`-`600px`. More complex layouts might benefit from doing a full sized layout above `1200px`, an altered "tablet" layout between `600px` and `1200px` and mobile below `600px`. The real takeaway here is that it doesn't really matter exactly where you set your breakpoints... just do what makes sense for your project.
+
+#### Zooming!
+Something to be aware of: in most browsers, zooming in on a webpage will change the effective resolution of that page. So if your browser windows is exactly `1000px` wide, zooming in will cause the page to behave as if the screen is _smaller_, and will trigger media-queries based on the simulated/zoomed screen resolution. Zooming _out_ can be handy for debugging issues that arise on screens that are larger than your actual computer screen... and forgetting that you've zoomed in or out on a webpage can cause some real confusion when breakpoints refuse to trigger at the correct points.
+
+### Print Styles
+You'll often see media queries defined with the `screen` keyword like so:
+
+~~~css
+@media screen and (max-width: 480px) {
+}
+~~~
+
+This is mostly unnecessary, but points toward another very useful capability of media queries: changing styles based on media-type. Everything we've covered so far has been specifically intended for viewing on some kind of screen so specifying `screen` is somewhat redundant. It is possible, however, to create a different set of styles for your website when it is sent to your printer or viewed in print-preview mode by using the `print` keyword.
+
+~~~css
+@media print {
+  /* print styles go here! */
+}
+~~~
+
+This is not something we're going to focus on in this curriculum, but in some cases it may be something you want to consider taking advantage of. It's fairly common to change some colors (i.e. make things black/white) and add `display: none` to hide elements that are useless in a printed environment (buttons, nav-links etc.).
+
+### Assignment
+<div class="lesson-content__panel" markdown="1">
+1. Look through [Using media queries](https://developer.mozilla.org/en-US/docs/Web/CSS/Media_Queries/Using_media_queries) on MDN. There are a few additional things you can do with media-queries that might be worth knowing about, though their usage is much less common.
+</div>
+
```

Comment:
Can we add an assignment here for the learner to make a simple layout with one media query on codepen? Just so they can get some practice writing one from scratch. Something like:

On codepen, create a very simple web page layout with a nav bar including some links to different pages. These links don't have to lead anywhere, but make them look like they do (About, Contact, Blog, etc.). Using what you've learned about media queries, make this nav bar turn into a [hamburger icon](https://fontawesome.com/v5.15/icons/bars?style=solid) when the screen width drops below 500px.

Reply:
yep.. that's planned, just not in this PR.  those will go in the css-exercises repo.